docs/developer-guide/coal-setup.md

```diff
@@ -346,6 +345,45 @@ Configuring CNAPI to allow headnode provisioning and over-provisioning (allow a
 Fabrics are not required to run CoaL. If you need to setup fabrics for
 development purposes you can refer to [Set up fabrics in CoaL](./coal-post-setup-fabrics.md)
 
+### Setup SDC-Docker
+
+SDC-Docker is not required to run CoaL. If you need to set up SDC-Docker with your CoaL, refer to the [Installation Instructions for SDC-Docker](https://github.com/joyent/sdc-docker#installation).
+
+### Configure Client Access to CloudAPI
+
+If you want to set up CLI on your laptop to access the CloudAPI on your CoaL, then follow the instructions for [Setting up CLI for CloudAPI](https://github.com/joyent/sdc-cloudapi/blob/master/docs/index.md#getting-started).
+
+But first, you need to perform a couple of steps on your headnode.
+
+1. Find the IP address of the CloudAPI zone by doing:
+
+   ```bash
+   [root@headnode (coal-1) ~]# sdc-vmapi /vms?alias=cloudapi | json -H -ga alias nics.1.ip
+   ```
+
+   For example, if you get the following returned values "cloudapi0 10.88.88.5", then the URL of your CloudAPI endpoint is <https://10.88.88.5>.
+
+2. Create a user:
+
+   ```bash
+   [root@headnode (coal-1) ~]# sdc-useradm create -A login={username} email={emailaddress} userpassword={password}
+   ```
+
+   Substiture real values for _{username}_, _{emailaddress}_ and _{password}_.
+
+When configuring CLI on your laptop, set the SDC_URL environment variable to <https://10.88.88.5>, and the SDC_ACCOUNT environment variable to _{username}_.  Also, remember to add the SSH key that you created to the _{username}_.  To do this:
+
+   ```bash
+   # Copy the SSH public key from your laptop to CoaL
+   scp ~/.ssh/id_rsa.pub root@10.88.88.200:/var/tmp/
+   
+   # ssh to the CoaL headnode
+   ssh root@10.88.88.200
+   
+   # Add the SSH key. Substiture real value for {username}.
+   sdc-useradm add-key {username} /var/tmp/id_rsa.pub
+   ```
+
```

Comment:
Lines 348 to 386: All prose should be hard wrapped prior to 80 columns for readability in terminals.